### PR TITLE
Sql

### PR DESCRIPTION
**Pull request by skripatch2**
I had some problems to import 3wifi.sql to a new database. First error was "Table 'base' doesn't have option". I found the solution that is to remove ROW_FORMAT=FIXED because the table was MyISAM but now is InnoDB. https://stackoverflow.com/questions/29486583/table-storage-engine-for-table-doesnt-have-this-option-on-order-by-query-err/32083169

Second error was invalid default value for `created` in `tasks` table. It is because mysql on my ubuntu server has enabled NO_ZERO_IN_DATE and NO_ZERO_DATE by default. With this options enabled '0000-00-00 00:00:00' is not valid default value. So I changed it to CURRENT_TIMESTAMP.

### CODE REVIEW

**Comment by skripatch2** [comment]
Here is a stack overflow post about NO_ZERO_DATE https://stackoverflow.com/questions/9192027/invalid-default-value-for-create-date-timestamp-field

File: 3wifi.sql [hunk 1]
```diff
@@ -36,7 +36,7 @@ CREATE TABLE `base` (
 	INDEX `ESSID` (`ESSID`),
 	INDEX `Time` (`time`),
 	UNIQUE INDEX `WIFI` (`NoBSSID`, `BSSID`, `ESSID`, `WiFiKey`, `WPSPIN`)
-) COLLATE='utf8_general_ci' ENGINE=InnoDB ROW_FORMAT=FIXED;
+) COLLATE='utf8_general_ci' ENGINE=InnoDB ;
```
Comment (binarymaster) [inline 3wifi.sql]: Everything looks ok!

I'd suggest to remove that trailing space between InnoDB and semicolon.

File: 3wifi.sql [hunk 2]
```diff
@@ -47,7 +47,7 @@ CREATE TABLE `geo` (
 	PRIMARY KEY (`BSSID`),
 	INDEX `quadkey` (`quadkey`),
 	INDEX `latitude` (`latitude`)
-) COLLATE='utf8_general_ci' ENGINE=InnoDB ROW_FORMAT=FIXED;
+) COLLATE='utf8_general_ci' ENGINE=InnoDB ;
```
Comment (binarymaster) [inline 3wifi.sql]: Ditto

File: 3wifi.sql [hunk 3]
```diff
@@ -137,7 +137,7 @@ CREATE TABLE `mem_base` (
 	INDEX `ESSID` (`ESSID`),
 	INDEX `Time` (`time`),
 	UNIQUE INDEX `WIFI` (`NoBSSID`, `BSSID`, `ESSID`, `WiFiKey`, `WPSPIN`)
-) COLLATE='utf8_general_ci' ENGINE=MEMORY ROW_FORMAT=FIXED;
+) COLLATE='utf8_general_ci' ENGINE=MEMORY ;
```
Comment (binarymaster) [inline 3wifi.sql]: Ditto

File: 3wifi.sql [hunk 4]
```diff
@@ -148,7 +148,7 @@ CREATE TABLE `mem_geo` (
 	PRIMARY KEY (`BSSID`),
 	INDEX `quadkey` (`quadkey`),
 	INDEX `latitude` (`latitude`)
-) COLLATE='utf8_general_ci' ENGINE=MEMORY ROW_FORMAT=FIXED;
+) COLLATE='utf8_general_ci' ENGINE=MEMORY ;
```
Comment (binarymaster) [inline 3wifi.sql]: Ditto

**Review by binarymaster — CHANGES_REQUESTED** [review]

**Comment by skripatch2** [comment]
I removed the spaces in the last commit

**Review by binarymaster — APPROVED** [review]

**Comment by binarymaster** [comment]
Many thanks!

**Pull request merged**